eleventy.js

```diff
@@ -14,7 +14,7 @@ module.exports = (eleventyConfig) => {
   eleventyConfig.addTransform(
     'eleventy-plugin-helmet',
     async (content, outputPath) => {
-      if (outputPath.endsWith('.html')) {
+      if (typeof outputPath === 'string' && outputPath.endsWith('.html')) {
```

Comment:
We can probably solve this using the optional chaining operator.
```suggestion
      if (outputPath.endsWith?.('.html')) {
```